it/uk/gov/hmrc/minorentityidentificationfrontend/assets/MessageLookup.scala

```diff
@@ -95,6 +95,17 @@ object MessageLookup {
     }
   }
 
+  object CaptureOfficePostcode {
+
+    val title = "What is the postcode used to register the association?"
+    val hint = "For example, AB1 2YZ"
+
+    object Error {
+      val invalid_office_postcode = "Enter a UK postcode"
```

Comment:
The names here do not match those in the messages file.

Reply:
updated thanks